packages/drivers/driver-web-cache/src/FluidCache.ts

```diff
@@ -117,10 +120,35 @@ export class FluidCache implements IPersistedCache {
 		});
 	}
 
+	private async openDb() {
+		if (this.db === undefined) {
+			this.db = await getFluidCacheIndexedDbInstance(this.logger);
```

Comment:
This could have a race condition - what if two callers call into openDB at the same time?
I think you should do something liek that (unless you do not what to deal with Deferred objects or Lazy promises):

```ts
const db = await getFluidCacheIndexedDbInstance(this.logger);
if (this.db == undefined) {
    this.db = db;
} else {
    db.close();
}
```